docs/basics/Epics.md

```diff
@@ -134,12 +134,14 @@ const fetchUserFulfilled = payload => ({ type: FETCH_USER_FULFILLED, payload });
 
 // epic
 const fetchUserEpic = action$ =>
-  action$.ofType(FETCH_USER)
+  action$
+    .ofType(FETCH_USER)
     .mergeMap(action =>
-      ajax.getJSON(`https://api.github.com/users/${action.payload}`)
+      ajax
+        .getJSON(`https://api.github.com/users/${action.payload}`)
```

Comment:
I'm not sure I like this opinion. I didn't care as much on the `ofType` examples but this one ends up being weird and not something I've seen anyone ever do. 🤔